tools/src/ExpoCLI.ts

```diff
@@ -17,7 +19,7 @@ export async function runExpoCliAsync(
   process.on('SIGINT', () => {});
   process.on('SIGTERM', () => {});
 
-  await spawnAsync('expo', [command, ...args], {
+  await spawnAsync('npx', ['expo', command, ...args], {
```

Comment:
Per @brentvatne , we should be using `create-expo-app` here and not `npx expo init`.